MODS.js

```diff
@@ -1008,9 +1013,6 @@ function doImport() {
 				}
 			}
 			
-			// creators (might be editors)
-			processCreators(host, newItem, "editor");
```

Comment:
You wrote that this is sometimes true, but not always. It looks unfortunate to delete these information completely. Can we save such persons more general as `"contributor"`s? Then, it would be much easier to correct it manually to an `"editor"` in the cases where it applies.

Can you give an example here again? (The url you provided does not work anymore.)

Reply:
Okay, this is the example: https://github.com/zotero/translators/pull/1488/commits/a595853955f7e9acfcb5c2710773dd79387f32b6#diff-20e6379d9c22904e95b3cb80adab4353R1940 . However, this change will also influence the other test cases negatively.

Note that the "editor" here is just the default value, but in most cases we have better information, e.g.
```xml
  <relatedItem type="host">
    <name type="personal">
      <namePart type="given">Ash</namePart>
      <namePart type="family">Amin</namePart>
      <role>
        <roleTerm type="text">editor</roleTerm>
      </role>
    </name>
   ....
```
This is examined in the `processCreator` function, also marc codes and only when this still leads to nowhere, the default value is taken: https://github.com/zotero/translators/blob/master/MODS.js#L772-L792

Reply:
Sorry I forgot to get back to you on this. The example is here: [MODS](https://api.nb.no/catalog/v1/metadata/URN:NBN:no-nb_digibok_2014012024006/mods) / [MARCXML](https://api.nb.no/catalog/v1/metadata/URN:NBN:no-nb_digibok_2014012024006/).

I can see the usefulness of including it if you're describing a chapter in an edited book, but would the title of the edited book be included as well then? You don't happen to have an example of a full mods record that demonstrates this?

Reply:
Here is a full example (it is one of the test cases):
```xml
<mods xmlns:xlink="http://www.w3.org/1999/xlink" version="3.4" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.loc.gov/mods/v3" xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-4.xsd">
	<titleInfo>
		<title>Fifth-Grade Boys' Decisions about Participation in Sports Activities</title>
	</titleInfo>
	<name type="personal">
		<namePart type="family">Conley</namePart>
		<namePart type="given">Alice</namePart>
		<role>
			<roleTerm type="text">author</roleTerm>
		</role>
	</name>
	<typeOfResource>text</typeOfResource>
	<relatedItem type="host">
		<titleInfo>
			<title>Non-subject-matter Outcomes of Schooling</title>
		</titleInfo>
		<name>
			<namePart>Good, Thomas L.</namePart>
			<role>
				<roleTerm>editor</roleTerm>
			</role>
		</name>
		<note type="statement of responsibility">ed. Thomas L. Good</note>
		<originInfo>
			<issuance>continuing</issuance>
		</originInfo>
		<part>
			<detail type="title">
				<title>Special issue, Elementary School Journal</title>
			</detail>
			<detail type="volume">
				<number>99</number>
			</detail>
			<detail type="issue">
				<number>5</number>
				<caption>no.</caption>
			</detail>
			<extent unit="pages">
				<start>131</start>
				<end>146</end>
			</extent>
			<date>1999</date>
		</part>
	</relatedItem>
</mods>
```

Reply:
Ouch, I was sure I had all the test passing :O  Ok, I'm fine with keeping it as it is now then!